### PR TITLE
Add documentation for nsinit

### LINKED ISSUE #468: Add basic documentation to the nsinit directory to spawn containers
**Problem**
nsinit lacks basic getting started documentation.

**Solution**

Add basic documentation to the `nsinit` directory that explains how to get a simple busybox container up and running and the available options that one can pass.

### PR DESCRIPTION
**Pull request by wonderflow**
About how to use and explaination of available options. Fixes #468 .

Signed-off-by: Sun Jianbo <wonderflow@zju.edu.cn>

cc @resouer

Hope to be helpful.

### CODE REVIEW

**Comment by wonderflow** [comment]
@dqminh  Fixed. Thank you!

File: nsinit/README.md [hunk 1]
```diff
@@ -65,3 +65,48 @@ You can identify if a process is running in a container by looking to see if
    
 You may also specify an alternate root directory from where the `container.json`
 file is read and where the `state.json` file will be saved.
+
+### How to use?
+
+Currently nsinit have 9 commands. Type `nsinit -h` to list all of them.
```
Comment (mrunalp) [inline nsinit/README.md]: s/have/has

File: nsinit/README.md [hunk 2]
```diff
@@ -65,3 +65,48 @@ You can identify if a process is running in a container by looking to see if
...
 You may also specify an alternate root directory from where the `container.json`
 file is read and where the `state.json` file will be saved.
+
+### How to use?
+
+Currently nsinit have 9 commands. Type `nsinit -h` to list all of them. 
+And for every alternative command, you can also use `--help` to get more 
+detailed help documents. For example, `nsinit config --help`.
+
+`nsinit` cli application is impelemented using [cli.go](https://github.com/codegangsta/cli).
```
Comment (mrunalp) [inline nsinit/README.md]: typo: impelemented --> implemented

File: nsinit/README.md [hunk 3]
```diff
@@ -65,3 +65,48 @@ You can identify if a process is running in a container by looking to see if
...
 file is read and where the `state.json` file will be saved.
+
+### How to use?
+
+Currently nsinit have 9 commands. Type `nsinit -h` to list all of them. 
+And for every alternative command, you can also use `--help` to get more 
+detailed help documents. For example, `nsinit config --help`.
+
+`nsinit` cli application is impelemented using [cli.go](https://github.com/codegangsta/cli). 
+Lots of details are done by cli.go, so the implementation of `nsinit` itself
```
Comment (mrunalp) [inline nsinit/README.md]: s/are done by/are handled in/

File: nsinit/README.md [hunk 4]
```diff
@@ -65,3 +65,48 @@ You can identify if a process is running in a container by looking to see if
...
+Lots of details are done by cli.go, so the implementation of `nsinit` itself 
+is very clean and clear.
+
+*   **config**	
+It will generate a standard configuration file for a container.  By default, it 
+will generate as the template file in [config.go](https://github.com/docker/libcontainer/blob/master/nsinit/config.go#L192). 
+It will modify the template if you have specified some configuration by options.
+*   **exec**	
+Start a container and execute a new command inside it. Besides common options, it
+have some special options as below.
```
Comment (mrunalp) [inline nsinit/README.md]: s/have/has

File: nsinit/README.md [hunk 5]
```diff
@@ -65,3 +65,48 @@ You can identify if a process is running in a container by looking to see if
...
+`nsinit` cli application is impelemented using [cli.go](https://github.com/codegangsta/cli). 
+Lots of details are done by cli.go, so the implementation of `nsinit` itself 
+is very clean and clear.
+
+*   **config**	
+It will generate a standard configuration file for a container.  By default, it 
+will generate as the template file in [config.go](https://github.com/docker/libcontainer/blob/master/nsinit/config.go#L192). 
+It will modify the template if you have specified some configuration by options.
+*   **exec**	
+Start a container and execute a new command inside it. Besides common options, it
```
Comment (mrunalp) [inline nsinit/README.md]: s/Start/Starts

**Comment by wonderflow** [comment]
I'm really appreciate for your check. Thank you! @mrunalp

**Comment by crosbymichael** [comment]
LGTM

**Comment by mrunalp** [comment]
LGTM

**Pull request merged**